### PR TITLE
[full-ci] [tests-only] Adjust drone env variables for changes to drone CI

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
Adjust the drone CI to take into account the changes made to env variables.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by phil-davis** [comment]
The PHP unit coverage stuff all works now, and SonarCloud has reported its result - good.

**Review by TheRealBro — APPROVED** [review]
LGTM although i'm not a drone.star expert

File: .drone.star [hunk 1]
```diff
@@ -1961,13 +1977,10 @@ def sonarAnalysis(ctx, phpVersion = DEFAULT_PHP_VERSION):
...
-                         "environment": {
-                             "MC_HOST_cache": {
-                                 "from_secret": "cache_s3_connection_url",
-                             },
-                         },
+                         "environment": MINIO_MC_ENV,
                          "commands": [
                              "mkdir -p results",
+                             "mc alias set cache $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY",
                              "mc mirror cache/cache/%s/%s/coverage results/" % (ctx.repo.slug, ctx.build.commit + "-${DRONE_BUILD_NUMBER}"),
```
Comment (xoxys) [inline .drone.star:1984]:
```suggestion
                             "mc mirror cache/$CACHE_BUCKET/%s/%s/coverage results/" % (ctx.repo.slug, ctx.build.commit + "-${DRONE_BUILD_NUMBER}"),
```

Right now `CACHE_BUCKET` is set but never used.

Comment (phil-davis) [reply .drone.star]: fixed

**Review by xoxys — APPROVED** [review]

**Pull request merged**